src/main/java/com/beerjournal/datamodel/entity/CollectableObjectEntity.java

```diff
@@ -0,0 +1,39 @@
+package com.beerjournal.datamodel.entity;
+
+import java.util.Optional;
+
+public abstract class CollectableObjectEntity {
+	protected Optional<String> id = Optional.empty();
```

Comment:
jeżeli entity jest tworzone przez serwis zewnętrzny to przed persystencją ma pusty optional
po akcji save jest on uzupełniany